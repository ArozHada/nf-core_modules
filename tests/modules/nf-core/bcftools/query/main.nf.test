nextflow_process {

    name "Test Process BCFTOOLS_QUERY"
    script "modules/nf-core/bcftools/query/main.nf"
    process "BCFTOOLS_QUERY"
    config "tests/modules/nf-core/bcftools_query/nextflow.config"
    tag "bcftools_query"

    test("Should run without failures") {

        when {
            params {
                outdir = "$outputDir"
            }
            process {
                """
                // define inputs of the process here. Example:
                // input[0] = file("test-file.txt")
                """
            }
        }

        then {
            // Assert that the process ran successfully
            assert process.success
            // Assert that the process produced the expected output
            assert snapshot(path(params.outdir).list()).match()
        }

    }

}
