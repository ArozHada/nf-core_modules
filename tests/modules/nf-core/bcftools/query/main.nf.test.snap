{
    "vcf_with_optional_files": {
        "content": [
            [
                "out.txt:md5,5a87e0865df2f0ab2884fc113ec2a70d",
                "versions.yml:md5,d1978d587f90107b0e03ea84fc59f930"
            ]
        ],
        "timestamp": "2023-04-10T22:05:36+0000"
    },
    "vcf": {
        "content": [
            [
                "out.txt:md5,c32a6d28f185822d8fe1eeb7e42ec155",
                "versions.yml:md5,d1978d587f90107b0e03ea84fc59f930"
            ]
        ],
        "timestamp": "2023-04-10T22:05:36+0000"
    }
}